docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     depends_on:
       - "db"
       - "memcached"
-    command: /badgr_server/manage.py runserver 0.0.0.0:8000
+    # command: /badgr_server/manage.py runserver 0.0.0.0:8000
```

Comment:
If I understand it correctly, this is now done via the `entrypoint.sh`? Then we also don't need it as a comment